helm/squidex/templates/_helpers.tpl

```diff
@@ -26,8 +26,8 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- define "squidex.mongoDefaultHostname" }}
   {{- $release := .Release}}
   {{- $clusterSuffix := .Values.clusterSuffix}}
-  {{- range $index, $i := until (int (index .Values "mongodb-replicaset").replicas) }}
-    {{- $replica := printf "%s-mongodb-replicaset-%d.%s-mongodb-replicaset.%s.svc.%s" $release.Name $i $release.Name $release.Namespace $clusterSuffix }}
+  {{- range $index, $i := until (int (index .Values "mongodb").replicaCount) }}
+    {{- $replica := printf "%s-mongodb-%d.%s-mongodb-headless.%s.svc.%s" $release.Name $i $release.Name $release.Namespace $clusterSuffix }}
```

Comment:
Why is this needed? Is this not a breaking change?

Reply:
The schema for the bitnami mongodb charts is different to the version that was deployed previously.

The change would require consumers to update their `values.yaml` files if they have made custom changes to their deployments, yes.

The new service that is deployed using the bitnami charts is appended with `-headless` and so requires an update to the connection string so that it can be addressed by DNS internally correctly. 
![image](https://user-images.githubusercontent.com/642269/203598545-6b66d64f-cea8-44a4-abec-96dec88deca6.png)

The other changes are based on how helm uses naming conventions to apply value substitution to child charts from the parent configuration, because the schema has changed on the child, we need to apply correct values.

Reply:
But then you would loose your previous data if you make an update, right? I am not sure what the convention for helm versioning is in this case.

Reply:
Helm versioning would really only apply here if you had a helm repository I think?

But yes, I think users wanting to deploy these changes to their cluster would need to potentially backup their data (they should be doing this anyway!!) and restore after applying this change.

I might have a look in the morning to see if there would be a way to keep the name of the `Persistent Volumes` and `Persistent Volume Claims` the same between these deployments (if they are not already), might take a little while as I will need to deploy `Squidex7`, add data to the CMS and then try and upgrade to this branch and see the outcome.

I will get back to you on this point and move the PR back into "Draft".